spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/tf/TensorflowLoader.scala

```diff
@@ -84,13 +87,32 @@ object TensorflowLoader{
   }
 
   /**
+   * Parse a tensorflow model protobuf file, read a list of op nodes from it
+   * @param graphProtoTxt where is the tf protobuf file
+   * @return
+   */
+  private[bigdl] def parseTxt(graphProtoTxt: String) : List[NodeDef] = {
```

Comment:
what's the difference from parse

Reply:
parseTxt process model.pbtxt, parse process model.pb